src/gsi/write_incr.f90

```diff
@@ -132,8 +132,8 @@ subroutine write_fv3_inc_ (grd,sp_a,filename,mype_out,gfs_bundle,ibin)
     integer(i_kind) :: ncid_out, lon_dimid, lat_dimid, lev_dimid, ilev_dimid
     integer(i_kind) :: lonvarid, latvarid, levvarid, pfullvarid, ilevvarid, &
                        hyaivarid, hybivarid, uvarid, vvarid, delpvarid, delzvarid, &
-                       tvarid, sphumvarid, liqwatvarid, o3varid, icvarid, &
-                       qrvarid, qsvarid, qgvarid
+                       tvarid, sphumvarid, o3varid!, liqwatvarid, icvarid, &
+                       !qrvarid, qsvarid, qgvarid
```

Comment:
@RussTreadon-NOAA @emilyhcliu I commented things out, but is that okay? Will it be better to remove the lines? We will just have to make a note of this when 16.3 is merged to the main branch.

Reply:
Since we do not intend to retain this change in develop, I think it's fine to simply comment out the variables and lines we do not want v16.3 to execute.